backend/src/main/kotlin/com/dinnye/backend/mapper/InfoDtoMapper.kt

```diff
@@ -59,6 +59,13 @@ interface InfoDtoMapper {
 
     fun map(parent: Parent, @MappingTarget dto: UserInfoDto)
 
+    fun map(user: User): UserInfoDto
+
+    //The return type User is an abstract class or interface. Provide a non abstract / non interface result type or a factory method.
+    //fun map(userInfoDto: UserInfoDto): User
+
+    fun map(userPostDto: UserPostDto, @MappingTarget doctor: Doctor)
```

Comment:
Ez ne itt legyen, inkább egy UserMapper-be tedd bele. De ott a MappingTarget lehet User, mivel jelenleg arra panaszkodik, hogy a User absztrakt és nem tudja létrehozni. De ha a usert paraméterként adod át akkor már tudja használni.